src/core/Link/Link.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+import { useStyles } from './styles';
+
+interface LinksProps {
+  pathname: string;
+  disabled?: boolean;
+  active?: boolean;
+}
+
+const Links: React.FC<LinksProps> = ({
+  pathname,
+  children,
+  disabled,
+  active,
```

Comment:
active shouldn't be something passed by the user...It's the property of the component

Reply:
@arkajyotiMukherjee I've done the active class exactly in the way that is in the example you sent  [](https://reactrouter.com/web/example/custom-link)